client/src/locales/en-US.json

```diff
@@ -912,34 +912,34 @@
             "email": "Email address:",
             "profile": "Profile choice:"
         },
-        "emptyList": "No user to display.",
-        "noMatch": "No users matching your filters.",
-        "listUsersFailed": "Failed to retrieve the users list. Please make sure you are online.",
-        "userSelectedCount": "No user selected | One user selected | {count} users selected",
+        "emptyList": "No members to display.",
+        "noMatch": "No members matching your filters.",
+        "listUsersFailed": "Failed to retrieve the organization members list. Please make sure you are online.",
+        "userSelectedCount": "No members selected | One member selected | {count} members selected",
         "userContextMenu": {
             "titleRemove": "Deletion",
-            "titleDetails": "User details",
-            "actionRevoke": "Revoke this user | Revoke these users",
+            "titleDetails": "Member details",
+            "actionRevoke": "Revoke this member | Revoke these members",
             "actionDetails": "View details",
             "titleAssignRoles": "Copy roles",
             "actionAssignRoles": "Copy workspace roles to...",
             "subtitleAssignRoles": "You can copy {sourceUser} roles to another person. Workspace roles will not be removed."
         },
         "greet": {
             "titles": {
-                "waitForGuest": "Onboard a new user",
+                "waitForGuest": "Onboard a new member",
```

Comment:
Not related to your change, but I thought we've dropped the term `Onboard`. shouldn't it be `Greet`?